exaudfclient/docs/script_options_requirments.md

```diff
@@ -0,0 +1,376 @@
+# System Requirement Specification
+
+This document outlines the user-centric requirements for the new Exasol UDF Client Script Options parser. It identifies the key features, high-level requirements, and user roles to ensure clarity and alignment with user needs.
+
+## Roles
+
+This section lists the roles that will interact with or benefit from the parser system.
+
+### Database Administrator
+Database Administrators manage the database environment and ensure the efficient execution of UDFs within Exasol, configuring and overseeing script execution.
+
+### Data Scientist
```

Comment:
Developer role potentially missing

Reply:
Hm, I'am not sure if we need a Developer Role. The Data Scientist role already covers the development of UDF's by customers.

Reply:
Developer is the person who integrates the lib into other software

Reply:
Added the role